netman/api/doc_config/api_samples/get_switch_hostname_vlans_vlan_interfaces.json

```diff
@@ -0,0 +1 @@
+[3000, 3001, 3002]
```

Comment:
Based on pull request #96 this is supposed to be a list of strings, the interfaces names, not a list of integers.